lang/en-EN/game.xml

```diff
@@ -2819,57 +2829,56 @@
-            <story_10><![CDATA[<b>Reached "Archmagus" rank:</b>⮀ This time around, you don't even bother with the front door of the College, and instead perform the magical equivalent of knocking on the door of the Council's pocket dimension. Once you feel their approval for you to enter, you smoothly step in and give a polite bow to the council as every eye in the cavernous room is pointed at you. After a quick introduction, you point your finger at one of the Archmagi seated just below the Council of Seven and challenge him for his title, to the shock of everyone present! The duel itself is gruelling, as much for you as for your opponent, but by the end he yields to you. You can barely stand as he takes his vividly purple cloak off and drapes it over your shoulders.]]></story_10>
-            <story_11><![CDATA[<b>Reached "Chair of the Council of the Seven" rank:</b>⮀ Time is precious this time, as you promptly step up to the council and challenge the Chair for the title. As is tradition for such a challenge, the other six members promptly rise and attack you together, but you endure the onslaught long enough to take them down one by one. While you are out of breath, the Chair gets up and calmly walks over to you, leaning in to whisper something in your ear: "Are you trying to take on the Seven?" You respond with only a nod, to which the Chair calmly takes off the bright white cloak with golden edges and puts it on your shoulders, the weight very nearly snapping your legs under you. "Then do not disappoint, and do not let the Seven wait," the former Chair says before walking out of the councilroom, leaving you to take the vacated seat of power.]]></story_11>
-            <!--Note: the stories kind of got away with me.-->
-            </wizard_college>
+            The wizard college is the finest school in all the realms. You're lucky to be afforded a chance to get in, although it will cost you a small fortune in tuition.<br>Take their tests and get a grade!<br>Can only have 1 Wizard College action.<br>Costs 500 gold and 10 favors.<br>Gives (all schools of magic skills) * (1 + main stat / 100) * sqrt(1 + times completed / 1000) * (original mana cost / actual mana cost) progress points per mana.<br>Unlocked at 60% of city toured.
+            ]]></tooltip>
+            <!-- Stories 1-11 have been replaced. Don't use those numbers or your stories may not show in the action log properly on old saves. -->
+            <story_12><![CDATA[<b>Attempted to attend:</b>⮀ Advancement in the Wizard College is determined by wizard duels. To rise to a title higher than Initiate, you must challenge someone at that rank and duel them for it. Every trick you can perform that they can't wins you a point, and vice versa. It looks like you'll need skills in all kinds of magic to succeed here.]]></story_12>
```

Comment:
I like that! I wrote that Initiates can go to classes but not really do anything else, since that fits in really nicely with the Student story (even if the rank titles don't really match up :P).